packages/plugins/blocks/blocks-aggrid/src/AgGridInput.js

```diff
@@ -29,6 +29,11 @@ const AgGridInput = ({ properties, methods, loading, events, value }) => {
 
   const memoDefaultColDef = useMemo(() => defaultColDef);
 
+  const getRowId = useCallback(
+    (params) => params.data[properties.rowId ?? '_id'] ?? JSON.stringify(params.data),
```

Comment:
An alternative implementation could be 
```js
params.data[properties.rowId] ?? params.data._id ?? params.data.id ?? JSON.stringify(params.data)
```
which also uses `id` since that is a more generic ID field than `_id` which is MongoDB specific?

Reply:
This makes sense. I updated this code in https://github.com/lowdefy/lowdefy/pull/1818/commits/51e45333b9b3ceff42bcbe3f94bcbb6710daf83f.